pkg/kapp/clusterapply/waiting_changes.go

```diff
@@ -119,7 +119,11 @@ func (c *WaitingChanges) WaitForAny() ([]WaitingChange, error) {
 		}
 
 		if time.Now().Sub(startTime) > c.opts.Timeout {
-			return nil, fmt.Errorf("Timed out waiting after %s", c.opts.Timeout)
+			trackedResourcesDesc := ""
```

Comment:
nit: trackedResourcesDesc => resourceNames

Reply:
nit: trackedResourcesDesc => timedOutResources

Reply:
Updated

Reply:
nit: hehe, i liked original trackedResourcesDesc (ideally would have been array) or resourceNames since these are names and not resources themselves.